packages/@react-types/shared/src/dnd.d.ts

```diff
@@ -142,10 +152,16 @@ interface DraggableCollectionEndEvent extends DragEndEvent {
 export type DragPreviewRenderer = (items: DragItem[], callback: (node: HTMLElement) => void) => void;
 
 export interface DraggableCollectionProps {
+  /** Hander that is called when a drag operation is started. */
   onDragStart?: (e: DraggableCollectionStartEvent) => void,
+  /** Hander that is called when the dragged element is moved. */
   onDragMove?: (e: DraggableCollectionMoveEvent) => void,
+  /** Hander that is called when the drag operation is ended, either as a result of a drop or a cancellation. */
   onDragEnd?: (e: DraggableCollectionEndEvent) => void,
+  /** A function that returns the drag item types for each item in the drag. */
```

Comment:
right, it returns an array of drag items, but each drag item has the following type:
https://github.com/adobe/react-spectrum/blob/3ac426e4a6b1fd0950fd17c765c2ca068d58f791/packages/%40react-types/shared/src/dnd.d.ts#L23-L25
hence why I said "drag item types" here. I'm happy to change it to "A function that returns the items being dragged" though since it seems to be more confusing the way I worded it